src/views/product-downloads-view/components/card-with-link/index.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Third-party imports
+import { ReactNode } from 'react'
+
+// Global imports
+import Card from 'components/card'
+import Text from 'components/text'
+
+// Local imports
+import classNames from 'classnames'
+import s from './card-with-link.module.css'
+
+interface CardWithLinkProps {
+	heading: string
+	subheading: string | ReactNode
+	icon?: ReactNode
+	link?: ReactNode
+	type?: 'info' | 'warning'
```

Comment:
added 'warning' here for the terms of use that has different styling than the info card